src/main/java/org/quiltmc/launchermeta/version_manifest/VersionEntry.java

```diff
@@ -15,19 +15,30 @@
  */
 package org.quiltmc.launchermeta.version_manifest;
 
+import java.util.Objects;
+import java.util.Optional;
+
 public class VersionEntry {
     private final String id;
     private final String type;
     private final String url;
     private final String time;
     private final String releaseTime;
+    private final String sha1;
+    private final int complianceLevel;
 
     public VersionEntry(String id, String type, String url, String time, String releaseTime) {
```

Comment:
Thoughts about marking this constructor as deprecated?

Reply:
I like that.